<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationWindow</class>
 <widget class="QWidget" name="AuthorizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="btnReg">
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
    background-color:  rgb(104,97,112);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color:  rgb(70, 68, 81);
}</string>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lePassword"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="leLogin"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Password">
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color: #A674FD&quot; &gt;Пароль&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Login">
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color: #A674FD&quot;&gt;Логин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignVCenter">
     <widget class="QPushButton" name="btnEnter">
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
    background-color:  rgb(104,97,112);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color:  rgb(70, 68, 81);
}</string>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; color: #e7e6ea&quot;&gt;Market Mentor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
